urbanaccess/gtfs/utils_format.py

```diff
@@ -426,43 +426,58 @@ def add_unique_agencyid(agency_df=None,stops_df=None,routes_df=None,trips_df=Non
+                                            agency_df=subset_agency_df)
             
-            trips_df = trips_agencyid(trips_df=trips_df,
-                                      routes_df=routes_df[['route_id', 'agency_id']],
-                                      agency_df=agency_df[['agency_id','agency_name']])
-
-            stops_df = stops_agencyid(stops_df=stops_df,
-                                      trips_df=trips_df[['trip_id', 'route_id']],
-                                      routes_df=routes_df[['route_id', 'agency_id']],
-                                      stop_times_df=stop_times_df[['trip_id', 'stop_id']],
-                                      agency_df=agency_df[['agency_id','agency_name']])
-
-            routes_df = routes_agencyid(routes_df=routes_df,
-                                        agency_df=agency_df[['agency_id','agency_name']])
-
-            stop_times_df = stop_times_agencyid(stop_times_df=stop_times_df,
-                                                routes_df=routes_df[['route_id', 'agency_id']],
-                                                trips_df=trips_df[['trip_id', 'route_id']],
-                                                agency_df=agency_df[['agency_id','agency_name']])
-
-            # TODO: It's obfuscatory to update the dataframe variables in such a deeply nested way.
-            #       Perhaps a more clear naming convention here before df_list is overridden
-            #       would make this steps intent more explicit.
+            trips_replacement_df = trips_agencyid(
+                                            trips_df=trips_df,
+                                            routes_df=subset_routes_df,
+                                            agency_df=subset_agency_df)
+
+            stops_replacement_df = stops_agencyid(
+                                            stops_df=stops_df,
+                                            trips_df=subset_trips_df,
+                                            routes_df=subset_routes_df,
+                                            stop_times_df=subset_stop_times_df,
+                                            agency_df=subset_agency_df)
+
+            routes_replacement_df = routes_agencyid(
+                                            routes_df=routes_df,
+                                            agency_df=subset_agency_df)
+
+            stop_times_replacement_df = stop_times_agencyid(
+                                            stop_times_df=stop_times_df,
+                                            routes_df=subset_routes_df,
+                                            trips_df=subset_trips_df,
+                                            agency_df=subset_agency_df)
 
             # need to update the df_list object with these new variable overrides
-            df_list = [stops_df,routes_df,trips_df,stop_times_df,calendar_df,calendar_dates_df]
+            df_list = [stops_replacement_df,
```

Comment:
Seems like the pythonic way would be to "not care" so might opt for the list since it removes an unnecessary step http://stackoverflow.com/questions/8159107/python-return-tuple-or-list